src/components/layout/BaseLayout.tsx

```diff
@@ -47,8 +48,12 @@ const BaseLayout: FC<PropsWithChildren<Props>> = ({
         {skyScraperAd ? (
           <chakra.aside
             display={{ '2xs': 'none', lg: 'block' }}
-            width={{ lg: '300px', xl: '500px' }}
-            minWidth={{ lg: '300px', xl: '500px' }}
+            css={{
+              '--lg-width': '300px',
+              '--xl-width': '500px',
+            }}
+            width={{ lg: 'var(--lg-width)', xl: 'var(--xl-width)' }}
+            minWidth={{ lg: 'var(--lg-width)', xl: 'var(--xl-width)' }}
```

Comment:
I'm wondering - would the conditional approach work for setting the CSS variable:

```suggestion
            css={{
              lg: { '--width': '300px' }, 
              xl: { '--width': '500px' },
            }}
            width="var(--width)"
            minWidth="var(--width)"
```
would be a bit more readable (assuming this approach works)

Reply:
This actually works, and I agree on the readability part.

Thanks!